tests/test_nwb_utils.py

```diff
@@ -0,0 +1,6 @@
+from ipfx.dataset.create import is_file_mies
+
+TEST_NWB_FILE_PATH = 'tests/data/nwb/Ctgf-T2A-dgCre;Ai14-495723.05.02.01.nwb'
+
+def test_is_file_mies():
+    assert is_file_mies(TEST_NWB_FILE_PATH) == True
```

Comment:
Can you add a test to test for the other type? I think it was HBG?

Reply:
There is no specific test/check for HBG files. The usage is checking if the file is MIES, and if not it assumes it's HBG.
There is one nwb file in the repo that is not MIES, I could create a test that shows that the NWB data loads successfully when `HBGNWBData` class is used?